src/Microsoft.AspNet.Owin/OwinFeatureCollection.cs

```diff
@@ -155,6 +155,16 @@ void IHttpResponseFeature.OnSendingHeaders(Action<object> callback, object state
             register(callback, state);
         }
 
+        void IHttpResponseFeature.OnResponseCompleted(Action<object> callback, object state)
```

Comment:
This? https://github.com/aspnet/HttpAbstractions/blob/08ddbe853130fe10bdde6f8b529e5a885e0974be/src/Microsoft.AspNet.Owin/OwinFeatureCollection.cs#L150-L158